pupil_src/launchables/player.py

```diff
@@ -186,7 +186,7 @@ def on_drop(window, count, paths):
         def get_dt():
             return next(tick)
 
-        pupil_data_path = os.path.join(rec_dir, "pupil_data")
+        pupil_data_path = os.path.join(rec_dir, "pupil_data.pldata")
```

Comment:
please revert the extension change for now.

Reply:
It is just for testing purposes. The PR is still WIP.